pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -237,6 +241,21 @@ type AutoscalerSpec struct {
 	PrometheusNamespace string `json:"prometheusNamespace,omitempty"`
 }
 
+// BucketLoggingSpec defines the configuration for the bucket
```

Comment:
change here to bucket logging configuration
```suggestion
// BucketLoggingSpec defines the bucket logging configuration
```

Reply:
Okay, Thanks